lib/completions/upload_encoder.rb

```diff
@@ -10,6 +10,14 @@ def self.encode(upload_ids:, max_pixels:)
           next if upload.blank?
           next if upload.width.to_i == 0 || upload.height.to_i == 0
 
+          desired_extension = upload.extension
+          desired_extension = "png" if upload.extension == "gif"
+          desired_extension = "png" if upload.extension == "webp"
+          desired_extension = "jpeg" if upload.extension == "jpg"
+
+          # this keeps it very simple format wise given everyone supports png and jpg
+          next if !%w[jpeg png].include?(desired_extension)
```

Comment:
should we log domething here?

Reply:
long term maybe, for now I think skipping obscure image formats is ok